homeassistant/components/withings/__init__.py

```diff
@@ -58,7 +58,7 @@
 
     hass.async_create_task(
         hass.config_entries.flow.async_init(
-            DOMAIN, context={"source": SOURCE_IMPORT}, data={}
+            DOMAIN, context={"source": SOURCE_USER}, data={}
```

Comment:
But this is wrong, it is an IMPORT.

Don't work around problems but instead we should solve them 😉  If the config flow aborts itself because it is already imported, it should clear the notification.

Reply:
The problem is even after the component has been configured, the notification is still present. How can I get the config flow to recognize it's done or just not show the notification at all?

Reply:
The notification is automatically removed if there are no integrations in progress that were automatically created. Do you abort the import ?

Reply:
Have been thinking about this more. We should just remove this whole import call.